pkgs/racket-doc/scribblings/reference/subprocess.scrbl

```diff
@@ -111,6 +111,19 @@ The @racket[subprocess] procedure returns four values:
 explicitly closed, usually with @racket[close-input-port] or
 @racket[close-output-port].
 
+Example:
```

Comment:
I would move all of these to the end of `@defproc` (right before `@history`).

Reply:
OK